tools/mfassignr/macros.xml

```diff
@@ -26,6 +36,85 @@
         </creator>
     </xml>
 
+    <xml name="kmdnoise_param">
+        <param name="input_file" type="data" format="tabular" label="Input data"
+               help= "Input data frame, first column is mass, second column is intensity"/>
+        <param name="upper_y" type="float" label="upper limit for the y intercept" value="0.2"
+               help= "The upper y-intercept value to isolate noise peaks in the equation for the KMD plot: y = 0.001123*x + b. Default value is set to 0.2, so that it does not interact with any potentially double-charged peaks."/>
+        <param name="lower_y" type="float" label="lower limit for the y intercept" value="0.05"
+               help="The lower y-intercept value to isolate noise peaks in the equation for the KMD plot: y = 0.001123*x + b. Default value is set to 0.05 to ensure no analyte peaks are incorporated into the noise estimation."/>
+        <param name="upper_x" optional="true" type="float" label="upper limit for the x intercept"
+               help="If not set, it defaults to maximum mass in the mass spectrum."/>
+        <param name="lower_x" optional="true" type="float" label="lower limit for the x intercept"
+               help="If not set, it defaults to minimum mass in the mass spectrum."/>
+    </xml>
+
+    <xml name="histnoise_param">
+        <param name="input_file" type="data" format="tabular" label="Input data"
+               help= "Input data frame, first column is mass, second column is intensity"/>
+        <param name="SN" type="float" label="signal-to-noise threshold" value="0"
+               help= "A numeric value for situations where a predefined noise value is desired, default is 0"/>
+        <param name="bin" type="float" label="bindwidth of the histogram" value="0.01"
+               help= "A numeric value determining the binwidth of the histogram, default is 0.01"/>
+    </xml>
+
+    <xml name="noise_threshold_params">
+       <param name="sn_ratio" type="float" label="SN ratio" value="1"
```

Comment:
value=1 but recommended is 6?

Reply:
Yeah - the tutorial recommends 6 but its not really explained anywhere why. Having a multiplier of 1 is like doing nothing, so seems neutral to me. If you think putting it by default to 6 is more intuitive we can change that.

Reply:
The value and the help is just not aligned. So it should match IMHO... And I guess I would take whatever the documentation says.